docker-compose.yml

```diff
@@ -1,16 +1,25 @@
-version: '3.1'
+version: "3.5"
 services:
-
-  hub-frontend-redis:
+  frontend:
+    image: quay.io/hmpps/prisoner-content-hub-frontend
+    build:
+      context: .
+    env_file:
+      - prisoner-content-hub-frontend.env
```

Comment:
Can't you just pass envs in when you start this up like redis is doing? It's how we normally do it, i.e. https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/docker-compose.yml We just pass env vars in.

Reply:
I.e. `--env-file ./config/.env.dev` option for starting up docker compose.

Reply:
Yeah can just pass them straight in.  The reason for having a separate file, is that on the backend we have separate files (so basically for consistency).  

I think it makes sense for the backend as it would make the `docker-compose.yml` file too large, considering there are 6 services in the [docker-compose.yml](https://github.com/ministryofjustice/prisoner-content-hub-backend/blob/5762a0c0981804fa054290d996620a6eda415c51/docker-compose.yml) file and about 30 env variables just for the backend https://github.com/ministryofjustice/prisoner-content-hub-backend/blob/5762a0c0981804fa054290d996620a6eda415c51/prisoner-content-hub-backend.env

Reply:
👍 understood, the docs have the env vars in anyways. Which is all fine, it doesn't actually change how it works for local dev if one chooses since you can pick and choose which services to start in the compose 🙂